lib/linters/color_variables.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const parser = require('postcss-values-parser');
+const util = require('util');
+
+module.exports = {
+    name: 'colorVariables',
+    nodeTypes: ['decl'],
+    message: '%s should be replaced with an existing variable.',
+
+    lint: function colorVariablesLinter (config, node) {
+        /**
+         * Just in case, since postcss will sometimes include the semicolon
+         * in declaration values
+         */
+        node.value = node.value.replace(/;$/, '');
+
+        const ast = parser(node.value, {
+            loose: true
+        }).parse();
+
+        // Parse the configuration for the style settings
+        if (config.style) {
+            throw new Error(`Invalid setting value for colorVariables: ${ config.style }`);
+        }
+
+        const results = [];
+        ast.first.walk((child) => {
+            // Only evaluate hex colors
+            if (child.type !== 'word' || !/^#/.test(child.value)) {
+                return;
+            }
+
+            // If this is a variable assignment, allow hex colors
+            const prop = node.prop;
+            if (/@[a-zA-Z0-9-_]*/.test(prop)) {
```

Comment:
Could you change this to the new [`isVariable`](https://github.com/lesshint/lesshint/blob/master/docs/developer-guide/api/utils.md#isvariable) util (I just merged it to master).